osbrain/tests/test_agent_transport.py

```diff
@@ -71,10 +74,12 @@ def test_agent_bind_given_address(nsproxy):
     """
     agent = run_agent('a0')
     # IPC
-    ipc_addr = str(uuid4())
-    address = agent.bind('PUSH', addr=ipc_addr, transport='ipc')
-    assert address.transport == 'ipc'
-    assert address.address.name == ipc_addr
+    if os.name == 'posix':
+        ipc_addr = str(uuid4())
+        address = agent.bind('PUSH', addr=ipc_addr, transport='ipc')
+        assert address.transport == 'ipc'
+        assert address.address.name == ipc_addr
+
```

Comment:
I'd say we split this test in two: one for IPC, one for TCP. Add then a skip-on-windows mark for the IPC test.

Reply:
I still think we should split this in 2.